File: servidor/routes/sugerencias.go
package routes

import (
	/*"proyectos/proyecto-leo/servidor/middleware"*/
	"proyectos/proyecto-leo/servidor/services"

	"github.com/gin-gonic/gin"
)

func SugerenciasRoutes(route *gin.Engine) {

	userGroup := route.Group("/sugerencias")
	{
		userGroup.GET("/find", services.ListarSugerencias)
		userGroup.DELETE("/delete/:id", services.EliminarSugerencia)
		userGroup.PUT("/update/:id", services.ActualizarSugerencia)
		userGroup.POST("/add", services.AgregarSugerencia)
	}
}

File: servidor-express/src/routes/sugerencias.js
const router = require('express').Router()
const sugerenciasController = require('../controllers/sugerenciasController')

router.get("/find", sugerenciasController.ListarSugerencias)
router.delete("/delete/:id", sugerenciasController.EliminarSugerencia)
router.put("/update/:id", sugerenciasController.ActualizarSugerencia)
router.post("/add", sugerenciasController.AgregarSugerencia)

module.exports = router